test/Microsoft.AspNet.Mvc.ViewFeatures.Test/ControllerTest.cs

```diff
@@ -1771,6 +1771,22 @@ public void TryValidateModelWithInvalidModelNoPrefix_ReturnsFalse()
         }
 
         [Fact]
+        public void TryValidateModel_Succeeds_WithoutValidatorMetadata()
+        {
+            // Arrange
+            var model = new TryValidateModelModel();
+            var binder = new GenericModelBinder();
```

Comment:
Very `true` that `ActionBindingContext` is fine w/ setting `ModelBinder` to `null` and the binder won't be used in this scenario. But I'm matching the OP's code in #3586.

The `IValueProvider` will also not be used.

The only real difference between this test and `TryValidateModelWithValidModel_ReturnsTrue` is we're using a real `IModelValidatorProvider` here. That test succeeded because the mock provider provided an empty array and never reached the product code I changed.

Are you suggesting I switch to `GetController(binder: null, provider: null)` since that's the minimal setup needed to hit the original issue?